ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -1174,8 +1174,20 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{app
         return from(services.applications.get(name, appNamespace))
             .pipe(
                 mergeMap(app => {
+                    // Check if this application is managed by another Argo CD instance
```

Comment:
I don't think this is the place to put this. It seems like application-resource-tree.tsx is more appropriate.